bioblend/_tests/TestGalaxyFolders.py

```diff
@@ -26,6 +26,12 @@ def test_show_folder(self):
         f2 = self.gi.folders.show_folder(self.folder['id'])
         self.assertEqual(f2['id'], self.folder['id'])
 
+    def test_show_folder_contents(self):
```

Comment:
Actually only some metadata key/value pairs were added in 16.01, I've changed the test and doc accordingly.